pkg/ceph/collectors/mon.go

```diff
@@ -298,10 +220,6 @@ func (m *MonitorCollector) Describe(ch chan<- *prometheus.Desc) {
 // Collect extracts the given metrics from the Monitors and sends it to the prometheus
 // channel.
 func (m *MonitorCollector) Collect(ch chan<- prometheus.Metric) {
-	if err := m.collect(); err != nil {
-		logger.Errorf("failed collecting monitor metrics: %+v", err)
-		return
-	}
 
 	for _, metric := range m.collectorList() {
```

Comment:
i believe these other statements are useless if we don't call collect above anymore